packages/components/src/components/FormGenerator/FormGenerator.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import {
+  FormGenerator,
+  FormGeneratorField,
+  FormGeneratorSupportedFields,
+} from '.';
+
+describe('<FormGenerator />', () => {
+  it('renders a FormGenerator', () => {
+    let lastnameField = 'lastName';
+
+    const fields: FormGeneratorField[] = [
+      {
+        component: FormGeneratorSupportedFields.TextInput,
+        props: {
+          label: lastnameField,
+          type: 'text',
+          name: lastnameField,
+          value: 'value',
+        },
+      },
+    ];
+    render(<FormGenerator fields={fields} stateOnChange={() => {}} />);
+    const textInput = screen.getAllByLabelText(lastnameField);
+    expect(textInput).toBeInTheDocument;
```

Comment:
Dette skal vel være et funksjonskall?
```suggestion
    expect(textInput).toBeInTheDocument();
```

Reply:
Opsi xD